### PR TITLE
Feat/add react hook form to reset form

### PR DESCRIPTION
**Pull request by vitor-afonso**
### PR description
- Updates reset page form to use react-hook-form
- Adds autoComplete to password fields in signup form

### CODE REVIEW

File: src/pages/ResetPage.js [hunk 1]
```diff
@@ -61,45 +61,67 @@ const ResetPage = () => {
...
-                error={passwordError}
-                autoFocus
+            <form noValidate onSubmit={handleSubmit(handleResetSubmit)}>
+              <Controller
+                name={componentProps.name.password}
+                control={control}
+                rules={{
+                  required: 'Password em falta',
+                  pattern: {
+                    value: /(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}/,
```
Comment (gustavosf2003) [inline src/pages/ResetPage.js:71]: You can create a global input setting file to write the patterns and avoid writing the same code twice for both the `ResetPage.js` and `EditProfilePage.js` files

File: src/pages/ResetPage.js [hunk 2]
```diff
@@ -61,45 +61,67 @@ const ResetPage = () => {
...
-        </>
+        </Box>
       )}
       {successMessage && (
         <Box sx={resetClasses.top}>
-          <Box sx={resetClasses.image}>
-            <img src={resetImage} alt='Forgot password' className='auth-images' />
-          </Box>
           <SuccessMessage message={successMessage} />
+          <Button onClick={() => navigate('/login')}>Entrar</Button>
```
Comment (gustavosf2003) [inline src/pages/ResetPage.js:124]: I would create a global file to store the route names and then just call `Routes.Login`

**Review by gustavosf2003 — APPROVED** [review]

File: src/pages/ResetPage.js (hunk 1 above)
Comment (vitor-afonso) [reply src/pages/ResetPage.js:71]: Yeah you are right! Gonna do that in another PR

File: src/pages/ResetPage.js (hunk 2 above)
Comment (vitor-afonso) [reply src/pages/ResetPage.js:124]: Gonna do that too 😉

**Pull request merged**